### PR TITLE
Add default en label in custom file button

### LINKED ISSUE #23612: File browser not working for non `en` language pages
Bootstrap 4
Chrome Versión 59 And Firefox Last Version
Ubuntu 16.04

File browser No Working
https://getbootstrap.com/docs/4.0/components/forms/#file-browser

Appears in white as in the following image
http://imgur.com/a/sPJgo

Do not see choose file or browse

### PR DESCRIPTION
**Pull request by chiraggmodi**
currently there is no default content added in custom file button. i have add English test as default test.

Fixes #23612

### CODE REVIEW

File: docs/4.0/layout/overview.md [hunk 1]
```diff
@@ -51,19 +51,21 @@ Since Bootstrap is developed to be mobile first, we use a handful of [media quer
 Bootstrap primarily uses the following media query ranges—or breakpoints—in our source Sass files for our layout, grid system, and components.
 
 {% highlight scss %}
-// Extra small devices (portrait phones, less than 576px)
-// No media query since this is the default in Bootstrap
+/****
```
Comment (XhmikosR) [inline docs/4.0/layout/overview.md]: Don't do this. You are sneaking in unrelated changes.

Comment (chiraggmodi) [inline docs/4.0/layout/overview.md]: sorry for that, have removed unrelated changes

**Comment by mdo** [comment]
Closing for #24259.
